City-of-Pori/pori-kada
# PORI-371 Hid the pori.fi English menu from page-not-translated page

To test:

1. drush fra -y && drush cc all
2. Go to https://local.visitpori.fi/en/page-not-translated and make sure that pori.fi English menu is not printed anywhere on the page.
